migration-engine/connectors/sql-migration-connector/src/sql_destructive_change_checker.rs

```diff
@@ -184,31 +184,15 @@ impl SqlMigrationConnector {
                             match type_change {
                                 Some(ColumnTypeChange::SafeCast) | None => (),
                                 Some(ColumnTypeChange::RiskyCast) => {
-                                    if self.features.contains(MigrationFeature::NativeTypes) {
-                                        plan.push_warning(
-                                            SqlMigrationWarningCheck::RiskyCast {
-                                                table: columns.previous().table().name().to_owned(),
-                                                column: columns.previous().name().to_owned(),
-                                                previous_type: columns
-                                                    .previous()
-                                                    .column_type()
-                                                    .full_data_type
-                                                    .to_string(),
-                                                next_type: columns.next().column_type().full_data_type.to_string(),
-                                            },
-                                            step_index,
-                                        );
-                                    } else {
-                                        plan.push_warning(
-                                            SqlMigrationWarningCheck::RiskyCast {
-                                                table: columns.previous().table().name().to_owned(),
-                                                column: columns.previous().name().to_owned(),
-                                                previous_type: format!("{:?}", columns.previous().column_type_family()),
-                                                next_type: format!("{:?}", columns.next().column_type_family()),
-                                            },
-                                            step_index,
-                                        );
-                                    }
+                                    plan.push_warning(
```

Comment:
I think this kept the wrong branch

Reply:
Ah in this case, the tests pointed out to the full_data_type not being populated, so we were rendering empty strings there. We can refine the dsiplay later, but now you at least have the column type family. So that's intended, but thanks for noticing!